app/src/main/java/com/endava/myendava/adapters/ChipsAdapter.java

```diff
@@ -1,41 +1,55 @@
 package com.endava.myendava.adapters;
 
 import android.content.Context;
+import android.content.Intent;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.EditText;
 import android.widget.TextView;
 
 import androidx.annotation.NonNull;
+import androidx.constraintlayout.widget.ConstraintLayout;
 import androidx.recyclerview.widget.RecyclerView;
 
-import com.endava.myendava.listeners.OnChipClickedListener;
 import com.endava.myendava.R;
-import com.endava.myendava.utils.TagColorManager;
+import com.endava.myendava.activities.FilteredTagsActivity;
+import com.endava.myendava.listeners.OnChipClickedListener;
+import com.endava.myendava.listeners.OnProfileEditedListener;
 import com.endava.myendava.models.Profile;
 import com.endava.myendava.models.Tag;
+import com.endava.myendava.utils.TagColorManager;
 import com.google.android.material.chip.Chip;
 import com.google.android.material.chip.ChipGroup;
 
 import java.util.List;
 import java.util.Map;
 
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
 public class ChipsAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
 
     private final static int TYPE_HEADER = 0;
     private static final int TYPE_ITEM = 1;
 
     private final Context context;
+    private final String email;
     private final Profile profile;
     private final Map<String, List<Tag>> tagsMap;
     private final OnChipClickedListener listener;
+    private final OnProfileEditedListener onEditListener;
```

Comment:
since you changed the name of the class ... could you also change the name of the variable also to onEdit**ed**Listener?